clr_loader/hostfxr.py

```diff
@@ -22,7 +22,11 @@ def __init__(self, runtime_config, dotnet_root=None):
                 raise RuntimeError("Can not determine dotnet root")
 
             try:
-                dotnet_path = os.readlink(dotnet_path)
+                dotnet_tmp_path = os.readlink(dotnet_path)
+                if os.path.isabs(dotnet_tmp_path):
+                    dotnet_path = dotnet_tmp_path
+                else:
+                    dotnet_path = os.path.abspath(os.path.join(os.path.dirname(dotnet_path), dotnet_tmp_path))
```

Comment:
Wouldn't simply doing `os.path.abspath(os.readlink(dotnet_path))` be enough?

Reply:
Thought about that. I'll try it later. Wasn't sure where abspath will base on. So I wanted to be sure

Reply:
So I tried that now and it didn't work. Because:

> 
>  os.path.abspath(path)
> 
>     Return a normalized absolutized version of the pathname path. On most platforms, this is equivalent to calling the function normpath() as follows: normpath(join(os.getcwd(), path))

So what abspath does is to take the current working directy (where you started the script) and apply ```normpath(join(os.getcwd(), path))``` .

That's why you would need all the fancy if else stuff. So in my case:
`dotnet_root` was `/usr/bin/dotnet` which was a symlink to `../share/dotnet/dotnet` that was being taken by the abspath and concatenated onto my user directory `/home/bneumann/pythonnet/clr-loader/share/dotnet/dotnet`